### PR TITLE
app-editors/vim,vim-core: Add accept_keywords for vim & vim-core

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# app-editors/vim,vim-core: Add accept_keywords for vim & vim-core

To be merged with https://github.com/flatcar-linux/portage-stable/pull/344

## Testing done

~Jenkins CI running: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/6101/cldsv/~

- [ ] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

File: profiles/coreos/targets/generic/make.defaults [hunk 1]
```diff
@@ -38,6 +38,7 @@ INSTALL_MASK="${INSTALL_MASK}
   /etc/sudoers
   /etc/wgetrc
   /etc/xinetd.d
+  /etc/vim/vimrc
```
Comment (krnowak) [inline profiles/coreos/targets/generic/make.defaults:41]: Possibly add /usr/share/vim/vim82/doc/gentoo-syntax.txt there too.

Comment (krnowak) [reply profiles/coreos/targets/generic/make.defaults:41]: And `/usr/share/vim/vim82/tools/vim132`, maybe with a comment about `gentoo-syntax` being a broken link, and vim132 using unsupported shell (`csh`).

**Comment by dongsupark** [comment]
This PR pulls in additional ebuilds like `app-vim/gentoo-syntax` into prod images, because `app-editors/vim-core` has its USE flag `-minimal`.
Would it be possible to avoid doing that by adding `app-editors/vim-core minimal` to `profiles/coreos/amd64/generic/package.use` and `profiles/coreos/arm64/generic/package.use`?

**Comment by sayanchowdhury** [comment]
Jenkins CI passed: http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/6302/cldsv/

File: profiles/coreos/arm64/generic/package.use [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # Disable crypt to avoid installing libsodlium in arm64.
 app-editors/vim		-crypt
+
+# minimal: Don't pull app-vim/gentoo-syntax in amd64
```
Comment (dongsupark) [inline profiles/coreos/arm64/generic/package.use:4]: s/amd64/arm64/

**Review by dongsupark — APPROVED** [review]
Looks good. Thanks.
vim-core occupies ~7MB in production images, and we could exclude some unnecessary files in theory. But we can do that later in another PR.

Nit below:

**Pull request merged**